storybook/stories/core/components/Input/AutocompleteInput.stories.js

```diff
@@ -5,18 +5,97 @@ import {
 } from '@orangehrm/oxd/core/components/Input/types';
 import {h, ref} from 'vue';
 import AutocompleteCustom from './AutocompleteCustom.story.vue';
-import QuickSearchWidget from './QuickSearchInput.story.vue';
+import AutoCompleteEvents from './AutoCompleteEvents.story.vue';
 
 export default {
   title: 'Inputs/AutocompleteInput',
   component: AutocompleteInput,
   argTypes: {
-    style: {control: {type: 'object'}},
-    hasError: {control: {type: 'boolean'}},
+    style: {
+      control: {type: 'object'},
+      table: {
+        type: {summary: 'Set custom style to the select'},
+      },
+    },
+    hasError: {
+      control: {type: 'boolean'},
+      table: {
+        type: {summary: 'Set error state to the select'},
+      },
+    },
     dropdownPosition: {
-      options: DROPDOWN_POSITIONS,
+      options: ['top', 'bottom'],
       defaultValue: BOTTOM,
     },
+    modelValue: {
+      control: {type: 'object'},
+      table: {
+        type: {summary: 'Set custom style to the select'},
+      },
+    },
+    'dropdown:closed': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when dropdown closed'},
+      },
+    },
+    'dropdown:blur': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when dropdown blur'},
+      },
+    },
+    'dropdown:clear': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when dropdown clear'},
+      },
+    },
+    'update:modelValue': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when select value updates'},
+      },
+    },
+    'select:enter': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when enter on selected value'},
+      },
+    },
+    'update:searchTerm': {
+      control: {type: 'array'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'emit event when search a value'},
+      },
+    },
+    beforeSelected: {
+      control: {type: 'object'},
+      defaultValue: [],
+      table: {
+        type: {summary: 'Attend a specific text to the start of the select'},
+      },
+    },
+    afterSelected: {
+      control: {type: 'object'},
```

Comment:
What does it means the type as object. It's not clear how to change it in the story book and get the afterSelected behaviour. Where will it appear?